gnuradio-runtime/lib/math/qa_fxpt_vco.cc

```diff
@@ -38,15 +38,15 @@ BOOST_AUTO_TEST_CASE(t0)
         input[i] = sin(double(i));
     }
 
-    for (int i = 0; i < SIN_COS_BLOCK_SIZE; i++) {
+    for (float i : input) {
```

Comment:
Pretty poor name for the float, now.

Maybe just `in`?

Reply:
agreed. This is just the autoconversion that clang-tidy does. When the original variable was short and the container ends in an "s", the clang-tidy pass tries to guess a singular and uses that (eg `for(int i = 0; i<inputs.size(); ++i)`->`for(auto& input : inputs)`), but here there's no plural-container :)